helper/sshutil/sshutil.go

```
@@ -129,6 +143,21 @@ func (sw *SSHSessionWrapper) RunCommand(ctx context.Context, cmd string) error {
 // - either a path to the private key file,
 // - or the content or this private key file
 func ReadPrivateKey(pk string) (ssh.AuthMethod, error) {
+	raw, err := ToPrivateKeyContent(pk)
```

Error is silently ignored....